src/main/java/com/autotune/analyzer/services/BulkService.java

```diff
@@ -86,7 +101,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
                 return;
             }
             jobDetails = jobStatusMap.get(jobID);
-            LOGGER.info("Job Status: " + jobDetails.getStatus());
+            LOGGER.info("Job Status: " + jobDetails.getSummary().getStatus());
```

Comment:
This should be:
`LOGGER.info("Job Status: {}",  jobDetails.getSummary().getStatus());`

Reply:
done